src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java

```diff
@@ -83,6 +84,7 @@ public ChannelRouteBuilder(@Value("#{'${dfspids}'.split(',')}") List<String> dfs
                                @Value("${bpmn.flows.mpesa-flow}") String mpesaFlow,
                                @Value("${rest.authorization.host}") String restAuthHost,
                                @Value("${operations.url}") String operationsUrl,
+                               @Value("${notification.enabled}") Boolean isNotificationServiceEnabled,
```

Comment:
Not a channel connector property. This is a workflow property. Since mpesa is the closest service representing the workflow, this zeebe variable should be set by MPesa service.

Reply:
if we don't set in channel connector then it will cause a loophole in the system. For example if the validation itself fails which is before mpesa service, hence the variable won't exist in zeebe and eventually will stuck at gateway.

Reply:
Also I asked @ankita10r to discuss it with you, weather to set in channel connector or mpesa. So just let me know.

Reply:
@danishjamal104   I see. Can we make it a sub property like mpesa.notification.enabled in channel connector then?

Reply:
yup, Il update this and https://github.com/fynarfin/ph-ee-env-template/pull/23 PR.